var-core/src/main/java/io/varhttp/VarServlet.java

```diff
@@ -1,26 +1,36 @@
 package io.varhttp;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 import java.util.function.Consumer;
+import javax.inject.Singleton;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+@Singleton
 public class VarServlet extends HttpServlet {
 	private final BaseVarConfigurationContext baseConfigurationContext;
 	Logger logger = LoggerFactory.getLogger(VarServlet.class);
 	private final ParameterHandler parameterHandler;
 	final ExecutionMap executions;
 	private ControllerMapper controllerMapper;
+	private List<VarWebSocket> webSockets = new ArrayList<>();
```

Comment:
final? and how is synchronization handled?

Reply:
This list is not populated, isn't the L:26 introduced to keep this  `private final RegisteredWebSockets registeredWebSockets;`?

Reply:
removed